demos/serverless/src/handlers.js

```diff
@@ -28,6 +31,14 @@ if(chimeSDKMeetingsEndpoint != 'https://service.chime.aws.amazon.com' && useChim
   chimeSDKMeetings.endpoint = new AWS.Endpoint(chimeSDKMeetingsEndpoint);
 }
 
+// Create an AWS SDK Media Pipelines object.
+// Use the MediaRegion property below in CreateMeeting to select the region
+// the meeting is hosted in.
+const chimeSdkMediaPipelines = new AWS.ChimeSDKMediaPipelines({ region: mediaPipelinesControlRegion });
```

Comment:
Can you move this construction just above `getClientForMediaCapturePipelines` function?